ocfweb/account/templates/account/register/index.html

```diff
@@ -41,21 +41,36 @@
                 {% endif %}
 
                 <p>
-                    This form is <strong>only</strong> for requesting
-                    individual accounts. If you want to request an account for
-                    a student group, please visit a staff member in the OCF lab
-                    during <a href="{% url 'staff-hours' %}">staff hours</a>.
+                    This form is used to request both individual and group accounts.
+                    Make sure you select the correct CalNet/CalLink identity to associate the new OCF account to.
+                    If your account already exists below, you should reset the password <a href="{% url 'change_password' %}">here</a> instead.
                 </p>
 
                 <hr />
-                <p><strong>Full Name</strong></p>
+                <p><strong>Full name of requestor</strong></p>
                 <p id="real-name">{{real_name|title}}</p>
+
+                {% if existing_accounts %}
+                <p><strong>Existing OCF user accounts</strong>
+                <p>{{existing_accounts|join:", "}}</p>
+                {% endif %}
+
+                {% if existing_group_accounts %}
+                <p><strong>Existing OCF group accounts</strong>
+                    {% for account in existing_group_accounts.values %}
+                        <p>{{account.name}}: {{account.accounts|join:", "}}</p>
+                    {% endfor %}
+                {% endif %}
+                <hr />
+
+                <p>You should select the identity that your new account will be associated with. This is either yourself or an RSO that you are a signatory for. Note that identities already associated to an OCF account will not show up here.</p>
+                <p id="account-association">{{form.account_association|bootstrap}}</p>
```

Comment:
This should be hidden for people that are not signatories.

Reply:
I actually think we should just leave it in. It lets users know that if they come back to this form they will have the option to select, and there will only be one option if they aren't a signatory. I'd like to hear your reasoning though.

Reply:
If it automatically selects your user when it's the only option, it's probably okay. We just need to keep in mind that normal user account creation is like two order of magnitudes more common than group account creation.